developer/src/kmc/src/kmc.ts

```diff
@@ -22,7 +22,8 @@ program
   .action((infile:any) => inputFilename = infile)
   .option('-d, --debug', 'Include debug information in output')
   .option('--no-compiler-version', 'Exclude compiler version metadata from output')
-  .option('-o, --out-file <filename>', 'where to save the resulting .kmx file');
+  .option('-o, --out-file <filename>', 'where to save the resulting .kmx file')
+  .option('-T, --test-data <filename>', 'Convert keyboard test .xml to .json');
```

Comment:
Note, we have [a project](https://docs.google.com/document/d/1ra7jQgg0ZDC8QcVIFRyA_Xw0kya3HS5ITSjJBsSNTtQ/edit#heading=h.me4lkdiw0zz5) to construct a good consistent command line interface for kmc so this may change in future.

kmc is still new and unreleased so we have the opportunity to do this right.

Building on experiences of other recent cmdline tools, e.g. ng, etc.